db/migrate/20231026234155_add_indeces.rb

```diff
@@ -0,0 +1,9 @@
+class AddIndeces < ActiveRecord::Migration[7.1]
+  def change
+    add_index :likes, :post_id
+    add_index :likes, :user_id
+    add_index :comments, :post_id
+    add_index :comments, :user_id
+    add_index :posts, :author_id
```

Comment:
- [ ] This line 7 is causing errors. Remember that in your posts table creation file you added the index already in line 13. Kindly remove this line here.

  ![image](https://github.com/chudisoft/RubyBlog/assets/36542986/b32ae5ba-9bbb-470e-a22d-ccdfc4d9c8a4)